Microsoft.Toolkit.Uwp.Notifications/Toasts/Builder/ToastContentBuilder.cs

```diff
@@ -0,0 +1,137 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+
+namespace Microsoft.Toolkit.Uwp.Notifications
+{
+#if !WINRT
+    /// <summary>
+    /// Builder class used to create <see cref="ToastContent"/>
+    /// </summary>
+    public partial class ToastContentBuilder
+    {
+        /// <summary>
+        /// Gets internal instance of <see cref="ToastContent"/>. This is equivalent to the call to <see cref="ToastContentBuilder.GetToastContent"/>.
+        /// </summary>
+        public ToastContent Content
```

Comment:
Why do you have a public attribute with a getter? Make this attribute private instead or remove the GetToastContent() method.

Reply:
It was discussed above:

>~~API currently use a public ToastContent (public getter, private setter) to build as we go along. Should we make this private ?~~ Decided to keep both. The method is there to support method chaining, while still leaving the getter for the case of extending via extension method.

Reply:
Ok thanks! You have my approval